xds/internal/xdsclient/xdslbregistry/xdslbregistry_test.go

```diff
@@ -69,6 +69,9 @@ func wrrLocalityBalancerConfig(childPolicy *internalserviceconfig.BalancerConfig
 }
 
 func (s) TestConvertToServiceConfigSuccess(t *testing.T) {
+	defer func(old bool) { envconfig.LeastRequestLB = old }(envconfig.LeastRequestLB)
+	envconfig.LeastRequestLB = true
```

Comment:
Same here, a test for expected behavior when the env var is set to false.

Reply:
Done.